components/playground/instance/tici.go

```diff
@@ -127,7 +127,7 @@ func (t *TiCIInstance) startMetaServer(ctx context.Context) error {
 	args := []string{}
 
 	// Use default or provided config path
-	metaConfigPath := filepath.Join(t.ticiDir, "ci", "test-meta.toml")
+	metaConfigPath := filepath.Join(t.ticiDir, "../../ci", "test-meta.toml")
```

Comment:
Why need to append "../../ci" to the "ticiDir"?
I think a better solution is provide different config file for metaservice and worker. For example --tici_meta_config --tici_worker_config

Reply:
If we do not add `--tici.config`, it will try to use default config under `tici/ci`.